src/app/app.py

```diff
@@ -95,12 +133,16 @@ async def get_schemas_metadata(survey_id: str) -> list[ReturnedSchemaMetadata]:
 
 @app.get("/v1/dataset_metadata", response_model=list[DatasetMetadata])
 async def get_dataset_metadata_collection(
-    survey_id: str, period_id: str
+    survey_id: str = "", period_id: str = ""
```

Comment:
Same issue as above, how come we're setting default values here?

Reply:
Changed empty string to None